cylc/flow/scripts/cat_log.py

```diff
@@ -367,28 +436,53 @@ def main(
     except KeyError:
         mode = options.mode
 
+    if tokens and tokens.get('cycle') and not tokens.get('task'):
+        print('Please provide a workflow, task or job ID', file=sys.stderr)
```

Comment:
Does this brink the task/job confusion back?
The id doesn't take a job identifier just a task ID?
```suggestion
        print('Please provide a workflow or a task ID', file=sys.stderr)
```

Reply:
No, `cat-log` accepts both tasks and jobs, what it doesn't accept are cycles because Cylc does not have logs for specific cycle points.